pkg/engine/daemon.go

```diff
@@ -172,7 +172,7 @@ func (e *Engine) startDaemon(_ context.Context, tool types.Tool) (string, error)
 		e.Ports.daemonWG.Done()
 	})
 
-	for i := 0; i < 20; i++ {
+	for i := 0; i < 120; i++ {
```

Comment:
Need to wait for more time to setup remote toolchain(like downloading runtime/pip install) before it can timeout

Reply:
@ibuildthecloud The problem is that today when we use clicky-chat to download knowledge-api, the clicky-chat daemon will timeout first before it can download the other runtime(python). Ideally this timeout could probably be higher to give more time to download.